Samples/WindowsCopilotRuntime/Contributing.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+This shows the project structure and how one can get started contributing to Windows Copilot Runtime (WCR) sample.
+
+## Project Structure
+
+The project follows the MVVM (Model-View-ViewModel) pattern. Here's how the code is organized:
+
+- **Models**: Contains the data models used in the application.
+  - `cs-winui/Models/ImageDescriptionModel.cs`
+  - `cs-winui/Models/ImageObjectExtractorModel.cs`
+- **ViewModels**: Contains the view models that handle the business logic and data binding for the views.
+  - `cs-winui/ViewModels/ImageDescriptionViewModel.cs`
+  - `cs-winui/ViewModels/mageObjectExtractorViewModel.cs`
+- **Pages**: Contains the XAML pages and their code-behind files. They represent the `Views` part of MVVM.
+  - `cs-winui/Pages/ImageDescriptionPage.xaml`
+  - `cs-winui/Pages/ImageDescriptionPage.xaml.cs`
+  - `cs-winui/Pages/ImageObjectExtractorPage.xaml`
+  - `cs-winui/Pages/ImageObjectExtractorPage.xaml.cs`
+- **Helpers**: Util, Converters and Controls folders contain common shared code used by all the pages
+  - `Controls` folder contains user controls for showing creating Session and example source codes
+  - `CodeBlockControl.xaml` and `CodeBlockControl.xaml.cs` are parts of user control which shows example source code for each api. The example code is fetched from `Examples` folder.
```

Comment:
Minor: I feel like a word or two is missing, but I'm not sure where.

"... are parts of users control**s** which show (no 's')"?
"... are part of **a** user control which shows"?

Reply:
we can ignore grammatical parts as long as they are clear enough for the user